### DISCUSSION TITLE
Artist page need option to filter out Albums by Various Artists or Albums that are Singles

### DISCUSSION THREAD

**Discussion by sdoowo200leldls** [music-assistant/support#2606] [Feature requests and ideas]
I would like the option to not see any of the items in the attached screenshot. I generally look for full albums by the artist.

![Screenshot_20240710-113729](https://github.com/music-assistant/hass-music-assistant/assets/106935585/3e8ae339-debe-436e-a836-45fa6cab8aab)

**Comment by OzGav** [comment]
You could use the favourite filter

**Comment by sdoowo200leldls** [comment]
Thanks for the reply! I usually go to an Artist's page to search through their albums; it's nearly impossible to see what they've solely released with all the "Various Artist" and "Single" albums listed. They would not be favorited as I've never listened to them before.

**Reply by OzGav** [reply]
Yep so when you get there favourite the ones you want to see and then turn on the filter

**Reply by sdoowo200leldls** [reply]
I'd probably just open Tidal and search there instead since they already segregate Artists albums vs EPs and Singles. But thanks for the suggestion!

**Reply by OzGav** [reply]
After further internal discussion there may be a solution which you will be happy with in future updates. Keep an eye out.

**Reply by sdoowo200leldls** [reply]
Great to hear! Thanks for looking at it more. I've fully replaced the use the Sonos app with Music Assistant and love it!

**Reply by OzGav** [reply]
This has been implemented in 2.1.0rc1. May need some more ui tweaks but it is working 
<img width="804" alt="image" src="https://github.com/user-attachments/assets/e4c1bf31-9f8b-4093-84d3-704cf0cf91b4">

**Reply by harryfine** [reply]
I am on 2.1.0rc2 and don't see that.  Here is my top bar.  I see that round disc on the left side, but there are no hints and it doesn't seem to do anything:

![2024-07-16_12-14](https://github.com/user-attachments/assets/8f7c10d9-2912-4367-98e6-275eb527f91a)

**Reply by OzGav** [reply]
You are looking at the album view. You need to look at the list of albums in the artist view.

**Comment by sdoowo200leldls** [comment]
I just installed the beta and this is exactly what I was looking for. Thank you so much!

**Comment by harryfine** [comment]
Cleared the MA cache and that did the trick.

However, there was no "tool tip" beside the circle as there is for the
other options on the toolbar.

Thanks so much

-----Original Message-----
From: OzGav ***@***.***>
Reply-To: music-assistant/hass-music-assistant
***@***.***>
To: music-assistant/hass-music-assistant
***@***.***>
Cc: harryfine ***@***.***>, Comment ***@***.***>
Subject: Re: [music-assistant/hass-music-assistant] Artist page need
option to filter out Albums by Various Artists or Albums that are
Singles (Discussion #2606)
Date: 2024-07-16 05:41:38 PM

Try clearing the browser cache
—
Reply to this email directly, view it on GitHub [1], or unsubscribe
[2].
You are receiving this because you commented.Message ID: <music-
assistant/hass-music-assistant/repo-
***@***.***>


[1] view it on GitHub
    https://github.com/orgs/music-assistant/discussions/2606#discussioncomment-10067003
[2] unsubscribe

**Reply by OzGav** [reply]
I'm seeing the tool tip "Album Type" with 2.1.0rc2?

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
